fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -316,8 +317,10 @@ private void setupLaunchFragment() {
     setCurrentFragment(R.id.fragment_launch, LaunchFragment::newInstance);
   }
 
-  private void setupConnectingFragment() {
-    setCurrentFragment(R.id.fragment_connecting, ConnectingFragment::newInstance);
+  private void setupConnectingActivity(String laoId) {
+    Intent intent = new Intent(this, ConnectingActivity.class);
+    intent.putExtra("LAO_ID", laoId);
```

Comment:
I generally try to avoid "magic values" as much as possible.
Especially when this is used as a key.

Could you create a constant for this ?

Reply:
Sure, I lazily followed was already there. I've updated everything